docs/src/modules/components/AppTableOfContents.js

```diff
@@ -129,6 +128,21 @@ function getItemsClient(items) {
   return itemsClient;
 }
 
+function useThrottledOnScroll(callback, delay) {
```

Comment:
It sounds like we might be able to get ride of react-event-listener :) https://bundlephobia.com/result?p=react-event-listener@0.6.6.